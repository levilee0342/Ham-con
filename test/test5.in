-29155
-3903
-28678
-22257
-11008
-26290
-21773
-2676
1430
-1368
-29231
-26780
-23822
-3292
-22801
-6245
-2595
-4872
-22028
-19382
