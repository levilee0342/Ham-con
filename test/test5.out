1430
-29231